public/index.html

```diff
@@ -1,54 +1,322 @@
-        <a title="فیلم های برگزیده" href="#">فیلم های برگزیده</a>
+  <body>
+    <header class="header_site">
+      <div class="bg_back_head">
+        <div class="bg_left_back"></div>
+        <div class="bg_right_back"></div>
       </div>
-      <div class="slider-container">
-        <button class="slick-prev">&lt</button>
-        <div class="slider-content-container">
-          <div class="slider-content">
-            <div class="slider-item">
-              <div class="slider-poster">
-                <a title="Movie Name" href="#">
-                  <img alt="inactive movie thumbnail" />
-                </a>
+      <div class="container">
+        <div class="rowflex">
+          <div class="left_side">
+
+            <div class="logo_head">
+              <a title="دیجی موویز" href="https://digimovie6.pro">
+                <img src="DGlogo.webp" alt="logo" title="دیجی موویز" />
+              </a>
+            </div>
+          
+
+          <div class="btns_change_mode" data-mode="">
+            <svg class="icon icon_moon">
+              <use xlink:href="#moon">
+                <symbol
+                  id="moon"
+                  xmlns="http://www.w3.org/2000/svg"
+                  viewBox="-12 0 448 448.045"
+                >
+                  <path d=""></path>
+                </symbol>
+              </use>
+            </svg>
+            <svg class="icon icon_sun">
+              <use xlink:href="#sun">
+                <symbol
+                  id="sun"
+                  xmlns="http://www.w3.org/2000/svg"
+                  viewBox="0 0 512 512"
+                >
+                  <circle cx="" cy="" r=""></circle>
+                  <path d=""></path>
+                </symbol>
+              </use>
+            </svg>
+            <div class="ball"></div>
+          </div>
+        </div>
+        
+        <div class="user_nav">
+          <a class="btn_header register_btn" href="#">
+            <svg class="icon icon_add_user">
+              <use xlink:href="#add_user">
+                <symbol id="add_user" xmlns="#" viewBox="#">
+                  <g xmlns="#">
+                    <path d="#"></path>
+                    <path d="#"></path>
+                    <path d="#"></path>
+                  </g>
+                </symbol>
+              </use>
+            </svg>
+            ثبت نام
+          </a>
+          <a class="btn_header login_btn" href="#">
+            <svg class="icon icon_sign_in">
+              <use xlink:href="#sign_in">
+                <symbol id="sign_in" xmls="#" viewBox="#">
+                  <path d="#"></path>
+                  <path d="#"></path>
+                </symbol>
+              </use>
+            </svg>
+            وارد شوید
+          </a>
+        </div>
+
+        <div class="top_menu">
+          <div class="container">
+            <div class="rowflex">
+              <div class="menu_right_header">
+                <ul class="menuTopHeaderDigi">
```

Comment:
```suggestion
                <ul class="top-header-menu">
```

`top-header-menu` would be a better name